api/src/main/java/io/druid/data/input/impl/StringInputRowParser.java

```diff
@@ -37,6 +38,7 @@
 
 /**
  */
+@PublicApi
 public class StringInputRowParser implements ByteBufferInputRowParser
```

Comment:
I was thinking it would be useful for people that want to implement extension firehoses that parse strings. Otherwise they'll need to duplicate a lot of logic.